doc/source/release-notes/Percona-Server-5.7.30-33.rst

```diff
@@ -0,0 +1,67 @@
+.. _PS-5.7.30-33:
+
+================================================================================
+*Percona Server for MySQL* 5.7.30-33
+================================================================================
+
+:Date: May 25, 2020
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.30-33
+includes all the features and bug fixes available in
+`MySQL 5.7.30 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-30.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-6951`: Documented new RocksDB variables: rocksdb_delete_cf, rocksdb_enable_iterate_bounds, and rocksdb_enable_remove_orphaned_dropped_cfs.
+* :jirabug:`PS-4464`: Exposed the last global transaction identifier (GTID) executed for a CONSISTENT SNAPSHOT.
+* :jirabug:`PS-6886`: Documented a new variable rocksdb_cache_dump which includes RocksDB block cache content in a core dump.
+* :jirabug:`PS-6910`: Documented the new RocksDB variable: rocksdb_stats_level.
+* :jirabug:`PS-6901`: Documented the new RocksDB variable: rocksdb_read_free_rpl.
+* :jirabug:`PS-6926`: Documented new RocksDB variables: rocksdb_table_stats_recalc_threshold_pct, rocksdb_table_stats_recalc_threshold_count, rocksdb_table_stats_background_thread_nice_value, rocksdb_table_stats_max_num_rows_scanned, rocksdb_table_stats_use_table_scan. rocksdb_table_stats_background_thread_nice_value,  rocksdb_table_stats_max_num_rows_scanned,  rocksdb_table_stats_use_table_scan, and rocksdb_trace_block_cache_access.
+* :jirabug:`PS-6891`: Documented the new RocksDB variable: rocksdb_master_skip_tx_api.
+* :jirabug:`PS-6885`: Documented the new variable rocksdb_rollback_on_timeout which allows the rollback of an entire transaction on timeout.
+* :jirabug:`PS-6890`: Documented the new RocksDB variable: rocksdb_blind_delete_primary_key.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7031`: add gcc-10 to platforms tested by Travis and Azure
+* :jirabug:`PS-6984`: Updated the zstd submodule to v1.4.4.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-6979`: If CREATE USER remains in the processlist, other users may see it. Modified the processing to call clean up functions as part of event. (Upstream :mysqlbug:`99200`)
```

Comment:
```suggestion
* :jirabug:`PS-6979`: CREATE USER statement could remain in the processlist after the statement has completed, causing other users to may see it. Modified the processing to call clean up functions as part of event cleanup. (Upstream :mysqlbug:`99200`)
```

Reply:
Changed to:
The CREATE USER statement could remain in the processlist after the statement has completed, allowing other users to may see it. Modified the processing to call clean up functions as part of event cleanup.